resources/windows/msi/service.wxs.erb

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+<!-- include the set of service files -->
+
+  <%- service_files.each do |service| -%>
+    <?include $(sys.SOURCEFILEDIR)include/<%=  service.path %>' ?>
```

Comment:
@jpogran We could just use a wildcard include - the specific files will be copied from the product directories (i.e. puppet-agent), so we may need some associated code in the generate_package_articfacts method to copy the puppet-agent service files to the working area.